demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -240,6 +242,50 @@ def verify_readme_image_paths(self) -> bool:
             return False
         return True
 
+    def verify_readme_relative_urls(self) -> bool:
+        """ Validate readme (not pack readme) relative urls.
+
+                Returns:
+                    bool: True If there are no invalid relative urls.
+                """
+        # If there are errors in one of the following validations return False
+        error_list = self.check_readme_relative_url_paths()
+        if len(error_list):
+            return False
+        return True
+
+    @error_codes('RM112')
+    def check_readme_relative_url_paths(self, is_pack_readme: bool = False) -> list:
+        """ Validate readme url relative paths.
+            prints an error if relative paths in README are found since they are not supported.
+
+        Arguments:
+            is_pack_readme (bool) - True if the the README file is a pack README, default: False
+
+        Returns:
+            list: List of the errors found
+        """
+        error_list = []
+        # If error was found, print it only if its not a pack readme. For pack readme, the PackUniqueFilesValidator
+        # class handles the errors and printing.
+        should_print_error = not is_pack_readme
+        relative_urls = re.findall(RELATIVE_MARKDOWN_URL_REGEX, self.readme_content,
+                                   re.IGNORECASE | re.MULTILINE)
+        relative_urls += re.findall(RELATIVE_HREF_URL_REGEX, self.readme_content,
+                                    re.IGNORECASE | re.MULTILINE)
+        for url in relative_urls:
+            # skip empty links
+            if not url[1]:
+                continue
+            # striping in case there are whitespaces at the beginning/ending of url.
+            error_message, error_code = Errors.invalid_readme_relative_url_error(url[1])
+            if error_code and error_message:  # error was found
+                formatted_error = self.handle_error(error_message, error_code, file_path=self.file_path,
```

Comment:
according to standard, we check `if self.handle_error(..):`, see other validations

Reply:
This behavior is because this function is also used in `pack_unique_files.py`, there we need to receive the errors and add them to the class object.
If you see any other way to achieve this I'll be happy to change it :)